README.md

```diff
@@ -1,5 +1,26 @@
-# alt-text-tracker
+# Alt text tracker
 
-The `alt.js` script runs every day to count the total number of images published to The Michigan Daily, and the number of images with alternative text. If a data file does not already exist, the script will retrieve counts since December 2022. Otherwise, the script will just retrieve data from within the last 24 hours, appending to the data file.
+Everything you need to build a Svelte project, powered by [`create-svelte`](https://github.com/sveltejs/kit/tree/main/packages/create-svelte).
 
-The script is generally accurate, though it does introduce mistakes from time to time. You can correct these mistakes by deleting the `data.csv` file and rerunning `alt.js`.
+## Developing
+
+Install dependencies with `pnpm install`, then to start a development server, run:
+
+```bash
+pnpm run dev
+
+# or start the server and open the app in a new browser tab
+pnpm run dev -- --open
+```
+
```

Comment:
Are there other set up steps that I should run? I installed dependencies and started the development server but I am getting a 500 error:

```
TypeError: Cannot read properties of undefined (reading 'map')
at alt-text-tracker/src/routes/+page.svelte:26:4
```

Reply:
Ah yeah this is outdated, because I use wrangler and a local db for locally testing the D1 binding. 

It's kind of convoluted, but I run the cron worker with `pnpm run worker-dev`, and go to http://localhost:8787/__scheduled to populate the db.

Then I run `pnpm run build`, then `pnpm run db-dev`.


I should probably just use the draft PR for the environment in the future. 💀

Reply:
Do I also have to run a migration script before running the worker script? I get a 500 at that URL.

Reply:
oh yeah the 000-init.sql script should be run as well sorry 😭